src/models/currentUser.js

```diff
@@ -51,6 +51,7 @@ export const newestMembership = currentUser =>
   maxBy(get('memberships', currentUser), 'created_at')
 
 export const hasFeature = (currentUser, key) => {
+  if (process.env.NODE_ENV === 'test') return true
```

Comment:
How do you feel about this Lawrence? Easier than setting them per test, but maybe you can see a reason not to do this...